sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLEnv.scala

```diff
@@ -51,6 +52,12 @@ private[hive] object SparkSQLEnv extends Logging {
       sparkContext.addSparkListener(new StatsReportListener())
       hiveContext = new HiveContext(sparkContext)
 
+      hiveContext.metadataHive.setOut(new PrintStream(System.out, true, "UTF-8"))
```

Comment:
nit: is UTF-8 the right thing here? It's not the default encoding on all platforms. Maybe omit that argument (which probably means using the platform default, even though the javadocs don't explicitly mention that)?